src/browser/declarativewebutils.cpp

```diff
@@ -92,6 +92,30 @@ void DeclarativeWebUtils::handleDumpMemoryInfoRequest(QString fileName)
     }
 }
 
+void DeclarativeWebUtils::openUrl(QString url)
+{
+
+    QFileInfo fileInfo(url);
```

Comment:
You could create this later only after checking the targetUrl.

Reply:
Not a big deal either as allocated in stack on variables will be free'ed after the call. Nevertheless, apparently left one extra empty line there.

Reply:
Main thing is that google.fi keeps on working which also looks like relative.